source/Calamari.Scripting/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Calamari.Common;
+using Calamari.Common.Plumbing.Logging;
+
+namespace Calamari.Scripting
+{
+    public class Program : CalamariFlavourProgram
```

Comment:
Should this be deriving from `CalamariFlavourProgramAsync`?

Reply:
Yup 😁

Reply:
well if we are making this a lib only for now, I'll remove the program.cs